docs/core-concepts/networking.md

```diff
@@ -28,20 +28,20 @@ Nodes and Containers attached to different Grids cannot communicate.
 
 ## Grid
 
-Each grid has a single overlay network. The default overlay network used is `10.81.0.0/19`, and it currently cannot be configured at creation time nor changed later.
+Each grid has a single overlay network. The default overlay network used is `10.81.0.0/16`, and it currently cannot be configured at creation time nor changed later.
 
-Each Grid may also include a set of Trusted Subnets, which are used for the Overlay Networking described further below.
+The grid's overlay network subnet is divided into two parts: `10.81.0.0/17` and `10.81.128.0/17`. The lower half is used by the Kontena Agent for statically allocated addresses used by infrastructure services such as etcd, and the upper half (`10.81.128.0 - 10.81.255.255`) is used for dynamically allocated service container addresses managed by the [Kontena IPAM](https://github.com/kontena/kontena-ipam).
 
-### IP Address Management
+Each Grid may also include a set of Trusted Subnets, which are used for the Overlay Networking between Host Node public and private IP addresses as described further below.
 
-Overlay network addresses are allocated by the Kontena Master, which tracks each allocated overlay network address.
-Using the default address allocation scheme, each Grid can contain up to 254 Host Nodes, and 8190 Service Containers.
+### IP Address Management
 
-When a Host Node's Kontena Agent is disconnected from the Kontena Master, the overlay network addresses used by the Node's Containers are released.
-Released overlay network addresses may be used for new Service Containers being deployed.
+Overlay network addresses are allocated by the [Kontena IPAM](https://github.com/kontena/kontena-ipam) service running on each host Node.
+Using the default address allocation scheme, each Grid can contain up to 254 Host Nodes, and 32768 Service Containers.
```

Comment:
With lower half of the default network I think you could have (theoretically) 32768 host nodes.